test/unit/test_runner_config.py

```diff
@@ -288,22 +282,21 @@ def test_generate_ansible_command_with_api_extravars():
     assert cmd == ['ansible-playbook', '-i', '/inventory', '-e', 'foo="bar"', 'main.yaml']
 
 
-@pytest.mark.parametrize('cmdline', [
-    '--tags foo --skip-tags'
-    '--limit "䉪ቒ칸ⱷ?噂폄蔆㪗輥"'
+@pytest.mark.parametrize('cmdline,tokens', [
+    (u'--tags foo --skip-tags', ['--tags', 'foo', '--skip-tags']),
+    (u'--limit "䉪ቒ칸ⱷ?噂폄蔆㪗輥"', ['--limit', '䉪ቒ칸ⱷ?噂폄蔆㪗輥']),
```

Comment:
It seems like the change made to this test was very subtle. Was this able to catch the bug before you made the patch?

Reply:
yep!